hazelcast-hibernate5/src/test/java/com/hazelcast/hibernate/CustomPropertiesTest.java

```diff
@@ -58,45 +48,6 @@
     @Rule
     public final ExpectedException exception = ExpectedException.none();
 
-    @Test
-    public void testNativeClient() throws Exception {
-        TestHazelcastFactory factory = new TestHazelcastFactory();
-        Config config = new ClasspathXmlConfig("hazelcast-custom.xml");
-        HazelcastInstance main = factory.newHazelcastInstance(config);
-        Properties props = getDefaultProperties();
-        props.remove(CacheEnvironment.CONFIG_FILE_PATH_LEGACY);
-        props.setProperty(Environment.CACHE_REGION_FACTORY, HazelcastCacheRegionFactory.class.getName());
-        props.setProperty(CacheEnvironment.USE_NATIVE_CLIENT, "true");
-        props.setProperty(CacheEnvironment.NATIVE_CLIENT_CLUSTER_NAME, "dev-custom");
-        props.setProperty(CacheEnvironment.CONFIG_FILE_PATH,"hazelcast-client-custom.xml");
```

Comment:
The properties except for `CONFIG_FILE_PATH`, which points to an empty client.xml, exist in `test-hibernate-client.cfg.xml` used in the new test class. Hence these are not set there.